src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -1040,6 +1040,63 @@ MulticopterAttitudeControl::task_main()
 					_controller_status_pub = orb_advertise(ORB_ID(mc_att_ctrl_status), &_controller_status);
 				}
 			}
+
+			if (_v_control_mode.flag_control_termination_enabled) {
```

Comment:
@anton-matosov The whole diff / PR makes sense, this is the only section I don't get yet entirely.

Are you trying to prevent an integrator windup here or is this needed to stop the props?